x-pack/test/cases_api_integration/security_and_spaces/tests/trial/internal/get_connectors.ts

```diff
@@ -413,7 +503,7 @@ export default ({ getService }: FtrProviderContext): void => {
 
           expect(Object.keys(connectors).length).to.be(1);
           expect(connectors[connector.id].id).to.be(connector.id);
-          expect(connectors[connector.id].needsToBePushed).to.be(false);
+          expect(connectors[connector.id].push.needsToBePushed).to.be(false);
         });
 
         it('sets needs to push to false the service now connector and true for jira', async () => {
```

Comment:
'sets needs to push to false *for the service now connector and true for jira'

What is the deal with the service now connector, it doesn't require/allow pushes?

Reply:
When we `createCaseWithConnector` we select the ServiceNow connector. Then we push in ServiceNow and then we select the Jira connector. The ServiceNow connector is already pushed so there is no need to push it again. On the other hand, Jira is not.